docs/en/user/user-documentation.md

```diff
@@ -0,0 +1,92 @@
+
+## Fields
+
+The list of Fields will be shown to a visitor as columns on the table, and there are a number of options available to the CMS editor to configure.
+
+![Column configuration](configureColumn.png)
+
+Each field can be configured as such:
+
+* The **Original label** is not editable, and serves to indicate the source of the data as coming from the CKAN resource.
+* The **Readable label** allows the label/name to be altered to be more readable if the one imported from CKAN is undesirable for any reason
+* If **Show In Results** is checked then this field will be shown as a column on the field site visitors will see. Otherwise it will not be shown. At least one field must be selected to be shown to the user.
+* If **Show in Detail** is checked, this field will be shown in the detail view that opens when a visitor clicks a row on the table shown to them. This is enabled for all fields by default.
+* If **Only show one value if duplicates exist** removes one of two (or more) results that have the same data in the result set shown to the website visitor
+* **Presented order** allows an editor to change the ordering of columns with more precision than the drag and drop provided on the GridField. E.g. it is possible to move a column between page of a GridField
+* The **Result conditions** allow for a "pre-filter" to be chosen to only show results matching (or not matching) a particular value.
+
+## Filters:
+The filters tab allows a CMS content editor to configure fields shown to a visitor that will allow them to search the results shown to them. One is always created by default after selecting a resource and saving the page. This default filter is a text type that searches all columns for matches to the term the visitor enters.
+
+By default there are two types of filters available to a content editor to configure, text and dropdown. The text filter allows a user to enter free text to filter results for. Dropdown type only allows pre-defined values to be chosen from.
+
+### Configuring a text filter:
+
+![Configuration options for a text filter](textFilterConfig.png)
+
+* A **Filter  label** can be given to indicate what type of search the content editor configures it to do.
+* **Column sources** allow the filter to be configured to either search either all columns, or if the checkbox is unchecked to search only a particular column or set of columns (multiple may e chosen).
+
+### Configuring the dropdown filter:
+
+This filter type has only one extra configuration from those provided by the text filter; the ability to supply options for the user to choose from.
+
+#### Options
+
+After choosing which columns the filter will apply to, the options selection will update to provide a list of all possible unique values those columns contain. The content editor can then choose to disable any of those options by un-checking the checkbox.
+
+![Dropdown filter configuration](dropdownFilterConfig.png)
```

Comment:
This screenshot looks dodgy in the first checkbox item - can we raise an issue for the label text not wrapping and/or adjust it in your inspector and re-take the screenshot?

Reply:
https://github.com/silverstripe/silverstripe-ckan-registry/issues/164